doc/classes/Input.xml

```diff
@@ -201,14 +201,16 @@
 				By default, the deadzone is automatically calculated from the average of the action deadzones. However, you can override the deadzone to be whatever you want (on the range of 0 to 1).
 			</description>
 		</method>
-		<method name="is_action_just_pressed" qualifiers="const">
+		<method name="is_action_just_pressed">
 			<return type="bool" />
 			<argument index="0" name="action" type="String" />
 			<argument index="1" name="exact" type="bool" default="false" />
+			<argument index="2" name="retire" type="bool" default="false" />
 			<description>
 				Returns [code]true[/code] when the user has [i]started[/i] pressing the action event in the current frame or physics tick. It will only return [code]true[/code] on the frame or tick that the user pressed down the button.
 				This is useful for code that needs to run only once when an action is pressed, instead of every frame while it's pressed.
 				If [code]exact[/code] is [code]false[/code], it ignores additional input modifiers for [InputEventKey] and [InputEventMouseButton] events, and the direction for [InputEventJoypadMotion] events.
+				If [code]retire[/code] is [code]true[/code], [method is_action_just_pressed] will no longer return [code]true[/code] for a pressed action for subsequent calls on the same frame or tick, unless the action is re-pressed.
```

Comment:
Simplifying wording slightly.
```suggestion
				If [code]retire[/code] is [code]true[/code], [method is_action_just_pressed] will no longer return [code]true[/code] for the given action when called again during the same frame or tick, unless the action is pressed again.
```

"unless the action is pressed again" is a bit... vague? I'm not sure what that means in this context, and I'm not sure the reader will, either.

Reply:
Yeah it's kind of hard to get across in a small number of words, because to some extent it depends on an explanation of how `just_pressed` works.

We can do some more revisions of the text if we decide to commit to this approach (as there are a number of ways of addressing the issue), and I'll make this change tomorrow.

A separate `retire_is_action_pressed()` function would be just as good, it just depends what we think will be easier for the user to understand.